pywbem_mock/_wbemconnection_mock.py

```diff
@@ -543,20 +545,94 @@ def compile_mof_string(self, mof_str, namespace=None, search_paths=None,
 
         mofcomp.compile_string(mof_str, namespace)
 
+    def compile_dmtf_classes(self, schema_mof_dir, schema_classnames,
+                             namespace=None, verbose=False):
```

Comment:
I would not name this method "compile_dmtf_classes()". It could be used on any CIM schema. How about "compile_schema_classes()"?

Reply:
DONE